sdk/java-sdk-testkit/src/main/java/kalix/javasdk/testkit/KalixTestKit.java

```diff
@@ -145,7 +154,9 @@ public Settings withAdvancedViews() {
   private final Settings settings;
 
   private boolean started = false;
-  private KalixProxyContainer proxyContainer;
+  private String proxyHost = "localhost";
+  private int proxyPort = 9000;
```

Comment:
More clear these defaults are only set in the local-proxy-scenario, since they are always overwritten with the values from test containers for the default case?

Reply:
Yes. Do you think it will be more clear not to set them with defaults?

Reply:
Yeah, that's what I meant, since it isn't really defaults for the "normal" scenario.

Reply:
done